bloomrun.js

```diff
@@ -16,6 +16,7 @@ function BloomRun (opts) {
   this._isDeep = opts && opts.indexing === 'depth'
   this._buckets = []
   this._properties = new Set()
+  this._defaultResult = null
```

Comment:
This needs to be an array, we can specify __multiple catchall__, as that is what the API says.
That's part of the reason I'm against having a catchall, as I think it's a feature of someone including bloomrun.